automated install/basic-install.sh

```diff
@@ -875,17 +892,28 @@ update_dialogs(){
                 echo "::: Running complete install script"
                 useUpdateVars=false
                 ;;
-	    esac
-	else
-		echo "::: Cancel selected. Exiting..."
-		exit 1
-	fi
+        esac
+    else
+        echo "::: Cancel selected. Exiting..."
+        exit 1
+    fi
 
 }
 
 main() {
+# Check arguments for the undocumented flags
```

Comment:
This has the probability of breaking the `set -e` check and masking any function failures instead of dumping to shell as it should. Checking this further....